CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+project(asio-rawsockets-example)
+cmake_minimum_required(VERSION 3.2)
+
+add_compile_options(-std=c++11)
+
+find_package(Boost REQUIRED system)
+
+INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIR})
+
+# Building server
```

Comment:
I'm not sure this example is really a server, I'd suggest just omitting this comment.

```suggestion
```

Reply:
yep agreed